src/adapters/index.ts

```diff
@@ -19,16 +20,12 @@ const messageKeyErrorParser = StrictSchema<MessageKeyError>()(
   })
 );
 
-export function getErrorMessage(error: unknown, prefixMsg?: string): string {
+export function parseError(error: unknown, prefixMsg?: string): string {
   if (typeof error === "string") {
     return prefixMsg ? `${prefixMsg}. ${error}` : error;
   } else if (error instanceof ZodError) {
-    const decodingIssues = error.errors
-      .map((issue) => `${issue.message} at path ${issue.path.join(".")}`)
-      .join(" | ");
-    return prefixMsg
-      ? `${prefixMsg}. Decoding issues: ${decodingIssues}`
-      : `Decoding issues: ${decodingIssues}`;
+    const decodingIssues = parseDecodingError(error).join(". ");
```

Comment:
What's the point of this function? I think that we are not going to get any extra information than a regular `JSON.stringify(error)` right?

Reply:
Human readability.

With this function you get this:
```
Decoding issue: Required at path node. Decoding issue: Expected object, received null at path network. Decoding issue: Expected number, received string at path recommendedFee.createAccount
```

Instead of this:
```
{"issues":[{"code":"invalid_type","expected":"object","received":"undefined","path":["node"],"message":"Required"},{"code":"invalid_type","expected":"object","received":"null","path":["network"],"message":"Expected object, received null"},{"code":"invalid_type","expected":"number","received":"string","path":["recommendedFee","createAccount"],"message":"Expected number, received string"}],"name":"ZodError"}
```

Reply:
But what's the point of having to maintain 50 lines of code just to add `Decoding issue` and `received`? For us, the developers (who are going to read that code) is better to have a JSON than that string in one line.

In that example I prefer to go to a JSON Formatter and in 5 seconds get this:

```js
{
   "issues":[
      {
         "code":"invalid_type",
         "expected":"object",
         "received":"undefined",
         "path":[
            "node"
         ],
         "message":"Required"
      },
      {
         "code":"invalid_type",
         "expected":"object",
         "received":"null",
         "path":[
            "network"
         ],
         "message":"Expected object, received null"
      },
      {
         "code":"invalid_type",
         "expected":"number",
         "received":"string",
         "path":[
            "recommendedFee",
            "createAccount"
         ],
         "message":"Expected number, received string"
      }
   ],
   "name":"ZodError"
}
```

Reply:
Let's wait for a third opinion on this one. I see the value of a human-readable msg but you clearly don't.

Reply:
@elias-garcia does not see the point of human-readable errors because as technical errors, they are always going to be interpreted by technicians. We are going to remove the function in charge of parsing the errors to a human-readable version and reporting only the stringified version of the error.

Reply:
Done